QAChat/Data_Processing/pdf_reader.py

```diff
@@ -31,7 +41,7 @@ def read_pdf(pdf_object, before: datetime, after: datetime, data_src: DataSource
     if len(text.strip()) < 100:
         text = __ocr_pdf(pdf_object)
 
-    return transform_to_data_information_list(text, before, after, data_src)
+    #return transform_to_data_information_list(text, before, after, data_src)
```

Comment:
```suggestion
    return transform_to_data_information_list(text, before, after, data_src)
```

If you really want to handle the Data yourself, just use your other method. But you don't have to use it, as described in the other comment.

Reply:
Overlooked this and inadvertently pushed it. But its now reverted.